src/components/Game/GameView.tsx

```diff
@@ -141,11 +184,13 @@ const GameView: React.FC<Props> = ({
   const didPlayNote = (note: number, playedBy: number) => {
     // TODO: update score
     console.log('Play', Tone.now() - delayedStartTime);
+    pressedNotes.current.add(note);
```

Comment:
this callback is trigged for both my notes, and my friend notes, in case u are not aware

Reply:
so u might want to store the playerId info as well

Reply:
ah yeah i didn't know that. Changed to only consider my own notes.